backend/src/tasks/LookingGlass.ts

```diff
@@ -0,0 +1,259 @@
+import {
+  Organization,
+  Domain,
+  Vulnerability,
+  connectToDatabase
+} from '../models';
+import getIps from './helpers/getIps';
+import { CommandOptions } from './ecs-client';
+import got from 'got';
+import { plainToClass } from 'class-transformer';
+import saveVulnerabilitiesToDb from './helpers/saveVulnerabilitiesToDb';
+import { exit } from 'process';
+import saveDomainsToDb from './helpers/saveDomainsToDb';
+
+async function getAuth() {
+  const results: any[] = await got
```

Comment:
We should create a type for results instead. Let's do:

```suggestion
  const results: LGResults[] = await got
```

And then define the interface LGResults elsewhere in the file. You can see this for an example: https://github.com/cisagov/crossfeed/blob/a70e552308996d7d396c101e331372122ab2fb0c/backend/src/tasks/censys.ts#L9-L19